src/tactic/interactive_expr.lean

```diff
@@ -65,6 +67,80 @@ meta def sf.of_eformat : eformat → sf
 | (of_format f) := sf.of_string $ format.to_string f
 | (compose x y) := sf.compose (sf.of_eformat x) (sf.of_eformat y)
 
+/-- `subtract x y` is some `z` when `∃ z, x = y ++ z` [fixme] this probably already exists. -/
+def subtract {α} [decidable_eq α]: (list α) → (list α) → option (list α)
```

Comment:
This is `list.get_rest`.

`#find` is a convenient way to find things like this:
```lean
import data.list.defs tactic.find

#find list _ → list _ → option (list _)
```